Wikipedia/Code/WMFArticlePreviewViewController.swift

```diff
@@ -67,4 +67,16 @@ open class WMFArticlePreviewViewController: UIViewController {
             self.view.layoutIfNeeded()
         }
     }
+    
+    public func apply(theme: Theme) {
+        guard viewIfLoaded != nil else {
+            return
+        }
+        titleTextColor = theme.colors.primaryText
```

Comment:
Shouldn't `WMFArticlePreviewViewController` have its own `theme` property that gets set if `apply:theme` is called before view loads (with `apply:theme` being called in `viewDidLoad` for those cases)?